core/src/stores/postgres.rs

```diff
@@ -433,6 +433,40 @@ impl Store for PostgresStore {
         Ok(runs_map)
     }
 
+    async fn delete_run(&self, project: &Project, run_id: &str) -> Result<()> {
+        let project_id = project.project_id();
+        let run_id = run_id.to_string();
+        let pool = self.pool.clone();
+        let mut c = pool.get().await?;
+        let tx = c.transaction().await?;
+
+        let create_proc = r#"
```

Comment:
FMI what is `r#""` ?

Reply:
Multiline string it's not r#"" but r#""# :)

Reply:
I think you can do other stuff like r#!""#! ? not sure.

Reply:
TIL 🙏

Reply:
I learned it from here: https://tourofrust.com/63_en.html 🦀